coding_setup.md

```diff
@@ -14,7 +14,7 @@ Squadbox uses Django with a MySQL backend (you can replace with any other backen
 * `/usr/bin/python2.7`
 * pip: `sudo easy_install pip`
 * `sudo pip install virtualenv `
-* create a virtualenv for this project: `virtualenv squadbox-env`
+* create a virtualenv for this project inside squadbox folder: `virtualenv --python=/usr/bin/python2.7 squadbox-env`
```

Comment:
I would say:

create a virtualenv for this project: `virtualenv squadbox-env`
If you have different versions of python on your computer (for instance, if you have anaconda installed), make sure you are using your system's python 2.7 by using the `python` flag to direct virtualenv to the right directory:
`virtualenv --python=/usr/bin/python2.7 squadbox-env`